pages/thc/[processRef]/[...slug].tsx

```diff
@@ -116,7 +118,16 @@ const ProcessPage: NextPage = () => {
         steps={innerSteps}
         manageStepTransitions={false}
         provideDatabase={false}
+        onIncompleteStep={(missingValues?: string[]): void => {
+          if (missingValues) {
+            // need to get real label of this, e.g. What type of ID, rather than e.g. `id-type` from component?
+            console.log(missingValues);
```

Comment:
To do this, I suggest adding an optional object to the `ProcessStepDefinition` that lets you provide error text for a value e.g. `errors: { required: { "id-type": "ID type is required" } }`.

Not required in this PR, though.